code/game/objects/structures/vehicles/lightcycle.dm

```diff
@@ -0,0 +1,252 @@
+/obj/item/key/lightcycle
+	name = "light rod"
+	desc = "A strange-looking glowing rod."
+	icon = 'icons/obj/vehicles.dmi'
+	icon_state = "lightcycle_keys_inactive"
+	w_class = W_CLASS_TINY
+	var/cycle_active = FALSE
+	var/obj/structure/bed/chair/vehicle/lightcycle/summoned_cycle = null
```

Comment:
Any reason you can't just replace any code checking `cycle_active` with whether or not `summoned_cycle` is null or not?